.github/workflows/go.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.18', '1.19']
+        go: ['1.18', '1.19', '1.20']
```

Comment:
```suggestion
        go: ['1.18', '1.19', '1.20', '1.21']
```

Reply:
@muhlemmer I commited my typo change already feel free to revert if you do not like them.

Should we also already cover v1.21 of go as well?